backend/service/chaos/experimentation/experimentstore/experiment_run.go

```diff
@@ -0,0 +1,93 @@
+package experimentstore
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
+
+	experimentation "github.com/lyft/clutch/backend/api/chaos/experimentation/v1"
+)
+
+type ExperimentRun struct {
+	id               uint64
+	startTime        time.Time
+	endTime          sql.NullTime
+	cancellationTime sql.NullTime
+	creationTime     time.Time
+}
+
+func (er *ExperimentRun) CreateProperties(now time.Time) ([]*experimentation.Property, error) {
+	status := timesToStatus(er.startTime, er.endTime, er.cancellationTime, now)
+	startTimeTimestamp, err := ptypes.TimestampProto(er.startTime)
+	if err != nil {
+		return nil, err
+	}
+
+	creationTimeTimestamp, err := ptypes.TimestampProto(er.creationTime)
+	if err != nil {
+		return nil, err
+	}
+
+	properties := []*experimentation.Property{
+		{
+			Id:    "run_identifier",
+			Label: "Run Identifier",
+			Value: &experimentation.Property_IntValue{IntValue: int64(er.id)},
+		},
+		{
+			Id:    "status",
+			Label: "Status",
+			Value: &experimentation.Property_StringValue{StringValue: statusToString(status)},
+		},
+		{
+			Id:    "run_creation_time",
+			Label: "Created At",
+			Value: &experimentation.Property_DateValue{DateValue: creationTimeTimestamp},
+		},
+		{
+			Id:    "start_time",
+			Label: "Start Time",
+			Value: &experimentation.Property_DateValue{DateValue: startTimeTimestamp},
+		},
+	}
+
+	var time sql.NullTime
+	if er.endTime.Valid {
+		time = er.endTime
+	} else if er.cancellationTime.Valid {
+		time = er.cancellationTime
+	}
+
+	endTimeTimestamp, err := TimeToPropertyDateValue(time)
+	if err != nil {
+		return nil, err
+	}
+
+	properties = append(properties, &experimentation.Property{
+		Id:    "end_time",
+		Label: "End Time",
+		Value: endTimeTimestamp,
+	})
+
+	cancelationTimeTimestamp, err := TimeToPropertyDateValue(er.cancellationTime)
+	if err != nil {
+		return nil, err
+	}
+
+	if status == experimentation.Experiment_STOPPED {
```

Comment:
nit: use switch and default case to handle unknown status.

Reply:
for unknown cases we just want to do nothing - not sure whether we need to have a switch's default for this. What do you think?

Reply:
Most of the time Ill just log in the event of an unknown type just so its observable.

Reply:
I agree in general but in this specific case other types are not unknown per se - we just do not want to create one extra property for them but logging an error/warning for this case would result in a non actionable noise.